clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1007,7 +1007,7 @@ class raw2trace_t : public trace_converter_t<raw2trace_t> {
     // Non-overridable parts of the interface expected by trace_converter_t.
     trace_entry_t *
     get_write_buffer(void *tls);
-    bool
+    virtual bool
```

Comment:
First, note that these are private.  The subclass will not be able to explicitly invoke these.  Is that ok?

Second, there is a separate raw2trace extension mechanism in place.  These functions are part of that mechanism,  the interface to trace_converter_t.  Would your extension of raw2trace fit better using trace_converter_t?  I'm guessing you've already evaluated that, but I can't tell from this PR.  Please provide further information on the goals and extent of the extensions and why this override approach was chosen instead of trace_converter_t.

Reply:
Forgot to move them to the protected section. Fixed.
No, trace_converter_t does not help here as I still need the functionalities in raw2trace.
I have added more details in the PR description.